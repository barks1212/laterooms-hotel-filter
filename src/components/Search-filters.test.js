import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });
import React from 'react';
import renderer from 'react-test-renderer';
import SearchFilters from './Search-filters';

describe('<SearchFilters>', () => {
  it('renders as expected without crashing and matches snapshot', () => {
    const tree = renderer.create(<SearchFilters filters={{temp: false}} filterHotels={jest.fn()} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});