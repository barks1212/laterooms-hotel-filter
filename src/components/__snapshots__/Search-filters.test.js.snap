// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchFilters> renders as expected without crashing and matches snapshot 1`] = `
<section
  className="filters"
>
  <section
    className="columns is-mobile is-gapless"
  >
    <section
      className="column is-one-half"
      id="columnOne"
    />
    <section
      className="column is-one-half"
      id="columnTwo"
    >
      <label
        className="checkbox"
        id="checkboxRight"
      >
        <input
          type="checkbox"
          value="temp"
        />
        <span
          className="is-size-7"
        >
          Temp
        </span>
      </label>
    </section>
  </section>
</section>
`;
